www/src/examples/Card/BgColor.js

```diff
@@ -1,97 +1,22 @@
-<>
-  <Card bg="primary" text="white" style={{ width: '18rem' }}>
+[
+  'primary',
+  'secondary',
+  'success',
+  'danger',
+  'warning',
+  'info',
+  'light',
+  'dark'
+].map((variant, idx) => (
+  <><Card bg={variant} text={variant=='light' ? 'dark' : 'white'}  style={{ width: '18rem' }}>
```

Comment:
here `idx` is unused variable, but it should be use as `key={idx}` for Card component. And for docs site, react-bootstrap is following some `strict` checks so you should replace `variant=='light'` with `variant==='light'`